src/audio/dai-zephyr.c

```diff
@@ -149,25 +150,25 @@ int dai_set_config(struct dai *dai, struct ipc_config_dai *common_config,
 	switch (common_config->type) {
 	case SOF_DAI_INTEL_SSP:
 		cfg.type = is_blob ? DAI_INTEL_SSP_NHLT : DAI_INTEL_SSP;
-		cfg_params = is_blob ? spec_config : &sof_cfg->ssp;
+		cfg_params = is_blob ? (void *)gtw_cfg : &sof_cfg->ssp;
```

Comment:
looks like we do this in lots of places, can we have a static inline to do this ?